tslint.json

```diff
@@ -1,20 +1,20 @@
 {
   "extends": ["tslint-config-prettier", "tslint-plugin-prettier"],
   "rules": {
+    "ban-comma-operator": true,
+    "jsdoc-format": [true, "check-multiline-start"],
     "no-duplicate-imports": true,
+    "no-duplicate-switch-case": true,
+    "no-duplicate-variable": [true, "check-parameters"],
     "no-invalid-template-strings": true,
     "no-object-literal-type-assertion": true,
-    "no-sparse-arrays": true,
-    "prefer-conditional-expression": true,
-    "prefer-object-spread": true,
-    "ban-comma-operator": true,
-    "no-duplicate-switch-case": true,
     "no-return-await": true,
-    "jsdoc-format": [true, "check-multiline-start"],
-    "no-duplicate-variable": [true, "check-parameters"],
+    "no-sparse-arrays": true,
     "no-this-assignment": true,
-    "space-within-parens": [true, 0],
+    "ordered-imports": true,
```

Comment:
Just added this.. and reordered the rules.

Reply:
👍